console/src/components/Services/ApiExplorer/GraphiQLWrapper/GraphiQLErrorBoundary.js

```diff
@@ -10,7 +11,7 @@ class GraphiQLErrorBoundary extends React.Component {
   componentDidCatch(error, info) {
     this.setState({ hasError: true, info: info });
     // most likely a localstorage issue
-    window.localStorage.clear();
+    clearLS();
```

Comment:
This would clear the entire local storage if there is a graphiQL error right? I think clearing whatever we use for graphIQL should be enough. Eg: clearing last stored SQL when there is a graphiQL error.
@rikinsk Please give your thoughts on the UX standpoint.  
cc: @beerose

Reply:
@beerose just a reminder that this is still an open question